rtgui/thumbnail.cc

```diff
@@ -217,8 +217,16 @@ rtengine::procparams::ProcParams* Thumbnail::createProcParamsForUpdate(bool retu
 
     const CacheImageData* cfs = getCacheImageData();
     Glib::ustring defaultPparamsPath = options.findProfilePath(defProf);
+    bool create = (!hasProcParams() || forceCPB);
 
-    if (defProf == DEFPROFILE_DYNAMIC && (!hasProcParams() || forceCPB) && cfs && cfs->exifValid) {
+    Glib::ustring outFName;
```

Comment:
Haha. :laughing: There are other ways to assign a conditional `const` value, but `?:` sure is the fastest.